tests/integration/datadictwithobjid/conftest.py

```diff
@@ -36,7 +36,6 @@ def get_parent(path):
 )
 
 
-@pytest.fixture(scope="session")
```

Comment:
I assume this is now the default?

Reply:
no, but the `pg_config` function is never used as a fixture, it's only used once as a function. The new version of pytest-flask protests that fixtures are not meant to be called